src/FFmpeg.cpp

```diff
@@ -116,7 +116,7 @@ class FindFFmpegDialog final : public wxDialogWrapper
    FindFFmpegDialog(wxWindow *parent, const wxString &path, const wxString &name)
        : wxDialogWrapper(parent, wxID_ANY, XO("Locate FFmpeg"))
        , mName(name)
-       , mFullPath(path, name)
+       , mFullPath(path, {})
```

Comment:
Reviewing the consequences of this change to mFullPath, I see the later call to SelectFile is affected.  Should you pass name there as the fourth argument, to fill in a field by default?  A small convenience for the user?

Reply:
I removed it on purpose. It no longer makes sense because it is a wildcard. On top of that, at least on windows, I observed that having a name messes up if you have more than one filter entry

Reply:
Ok